packages/common/client-utils/api-extractor.json

```diff
@@ -0,0 +1,4 @@
+{
+	"$schema": "https://developer.microsoft.com/json-schemas/api-extractor/v7/api-extractor.schema.json",
+	"extends": "@fluidframework/build-common/api-extractor-common-report.json"
```

Comment:
Since we're asking devs to explicitly annotate items as `@internal`, it might be good to add this rule here to enforce the presence of release tags on exported members.
```json
	"ae-missing-release-tag": {
				"logLevel": "error",
				"addToApiReportFile": false // Fail build rather than adding warning comment to API report
			},
```

Reply:
And if the base config doesn't include it, this one:
```json
"ae-incompatible-release-tags": {
				"logLevel": "error",
				"addToApiReportFile": false // Fail build rather than adding warning comment to API report
			}
```

Reply:
Good point. I'm curious what other reviewers think about this, so I'll leave this out change for now. If we end up merging this with that requirement I'll add this config.